apps/teams-test-app/src/pages/TeamsTestApp.tsx

```diff
@@ -0,0 +1,138 @@
+import TeamsAPIs from '../components/privateApis/TeamsAPIs';
+import VideoExAPIs from '../components/privateApis/VideoEffectsExAPIs';
+import ProfileAPIs from '../components/ProfileAPIs';
+import RemoteCameraAPIs from '../components/RemoteCameraAPIs';
+import SearchAPIs from '../components/SearchAPIs';
+import SecondaryBrowserAPIs from '../components/SecondaryBrowserAPIs';
+import SharingAPIs from '../components/SharingAPIs';
+import StageViewAPIs from '../components/StageViewAPIs';
+import TeamsCoreAPIs from '../components/TeamsCoreAPIs';
+import ThirdPartyCloudStorageAPIs from '../components/ThirdPartyCloudStorageAPIs';
+import CookieAccessComponent from '../components/ThirdPatryCookies';
+import Version from '../components/Version';
+import VideoAPIs from '../components/VideoEffectsApis';
+import VisualMediaAPIs from '../components/VisualMediaAPIs';
+import WebStorageAPIs from '../components/WebStorageAPIs';
+
+const TeamsTestApp: React.FC = () => {
+  const dialogWindowRef = React.useRef<IAppWindow | null>(null);
+
+  return (
+    <>
+      <div className="App-container">
+        <AppAPIs />
+        <AppInitializationAPIs />
+        <AppInstallDialogAPIs />
+        <AuthenticationAPIs />
+        <AppEntityAPIs />
+        <BarCodeAPIs />
+        <CalendarAPIs />
+        <CallAPIs />
+        <ChatAPIs />
+        <ClipboardAPIs />
+        <CookieAccessComponent />
+        <CustomAPIs />
+        <DialogAPIs />
+        <DialogCardAPIs />
+        <DialogCardBotAPIs />
+        <DialogUpdateAPIs />
+        <DialogUrlAPIs childWindowRef={dialogWindowRef} />
+        <DialogUrlBotAPIs />
+        <DialogUrlParentCommunicationAPIs childWindowRef={dialogWindowRef} />
+        <ExternalAppAuthenticationAPIs />
+        <ExternalAppCardActionsAPIs />
+        <ExternalAppCommandsAPIs />
+        <FilesAPIs />
+        <FullTrustAPIs />
+        <GeoLocationAPIs />
+        <Links />
+        <LocationAPIs />
+        <LogAPIs />
+        <MailAPIs />
+        <MarketplaceAPIs />
+        <MediaAPIs />
+        <MeetingAPIs />
+        <MeetingRoomAPIs />
+        <MenusAPIs />
+        <MessageChannelAPIs />
+        <MonetizationAPIs />
+        <NestedAppAuthAPIs />
+        <NotificationAPIs />
+        <OtherAppStateChangedAPIs />
+        <PagesAPIs />
+        <PagesAppButtonAPIs />
+        <PagesBackStackAPIs />
+        <PagesConfigAPIs />
+        <PagesCurrentAppAPIs />
+        <PagesTabsAPIs />
+        <PeopleAPIs />
+        <PrivateAPIs />
+        <ProfileAPIs />
+        <RemoteCameraAPIs />
+        <SearchAPIs />
+        <SecondaryBrowserAPIs />
+        <SharingAPIs />
+        <WebStorageAPIs />
+        <StageViewAPIs />
+        <TeamsCoreAPIs />
+        <TeamsAPIs />
+        <ThirdPartyCloudStorageAPIs />
+        <VideoAPIs />
+        <VideoExAPIs />
+        <VisualMediaAPIs />
+      </div>
+      <Version />
+    </>
+  );
+};
+
+export default TeamsTestApp;
```

Comment:
Is there a reason to prefer exporting this as a default export rather than just directly exporting the component? While it probably doesn't matter in this usage I think I recall something about default exports not being ideal if they can be avoided, e.g.,

[https://basarat.gitbook.io/typescript/main-1/defaultisbad](https://basarat.gitbook.io/typescript/main-1/defaultisbad)
and
[https://medium.com/@hizacharylee/why-you-should-avoid-default-exports-in-javascript-modules-7d087dde7ac6](https://medium.com/@hizacharylee/why-you-should-avoid-default-exports-in-javascript-modules-7d087dde7ac6)

...for some examples.